src/styles/main.css

```diff
@@ -501,28 +636,47 @@
 }
 
 .swagger-ui .parameters .parameters-col_name .parameter__in {
-  font-size: 13px;
+  font-size: var(--type-sm);
   font-family: var(--font-default);
   font-style: normal;
   font-weight: normal;
-  color: var(--black-400);
+  color: var(--text_colors-secondary);
 }
 
 .swagger-ui .parameters .parameter__deprecated {
-  font-size: 13px;
+  font-size: var(--type-md);
 }
 
 .swagger-ui .parameters select {
   width: 100%;
 }
 
+.swagger-ui .parameters input[type="email"]:disabled,
+.swagger-ui .parameters input[type="password"]:disabled,
+.swagger-ui .parameters input[type="search"]:disabled,
+.swagger-ui .parameters input[type="text"]:disabled,
+.swagger-ui .parameters textarea:disabled {
+  cursor: not-allowed;
+  font-style: italic;
+}
+
 .swagger-ui .parameters input[type="email"],
 .swagger-ui .parameters input[type="password"],
 .swagger-ui .parameters input[type="search"],
 .swagger-ui .parameters input[type="text"],
 .swagger-ui .parameters textarea {
+  display: block;
+  color: var(--input-color);
+  font-size: var(--input-font-size);
+  font-weight: var(--input-font-weight);
+  line-height: 24px;
+  border: none;
+  border-radius: 3px;
+  background-color: var(--input-background);
+  box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300)) !important;
```

Comment:
Please do no use and overwrite kongponents variables

If needed - please create a new variable and set value to corresponding value